### ISSUE TITLE
readme.txt Screenshots 

### ISSUE THREAD

**Issue by machine-user-mcc** [YahnisElsts/plugin-update-checker#313]
Hi There, thanks for the great work you've done here. It's very handy. 

Just want to know if screenshots should work on a gitHub private repo hosted plugin?

I added /assets directory and put the screenshots in there, then used an ordered list in the readme markup as outlined in the wordpress dev pages, but they don't work. 
Are they supposed to work? 
If so, can you provide an example of the exact syntax and setup for use with GitHub private repo?

Cheers, 
mike

**Comment by YahnisElsts** [comment]
No, that won't work. The way that screenshots work in `readme.txt` is a server-side feature of `wordpress.org`. GitHub doesn't support the same screenshot syntax. Also, at this time I have no plans to implement client-side support for that screenshot syntax.

Maybe you could use Markdown to insert the images instead? For example:
```markdown
![Alt text](https://octodex.github.com/images/yaktocat.png)
```

I'm not sure if that would work, but it seems worth a try.

**Comment by machine-user-mcc** [comment]
Ok, thanks. That's cool. I just wasn't sure if it was supposed to or not. I can live without it.
